ddev/src/ddev/cli/validate/licenses.py

```diff
@@ -0,0 +1,49 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import click
+from packaging.requirements import Requirement
+
+if TYPE_CHECKING:
+    from ddev.cli.application import Application
+
+
+@click.command(short_help='Validate third-party license list')
+@click.option('--sync', '-s', is_flag=True, help='Generate the `LICENSE-3rdparty.csv` file')
+@click.pass_context
+def licenses(ctx: click.Context, sync):
+    app: Application = ctx.obj
+    validation_tracker = app.create_validation_tracker('Licenses')
+
+    # Validate that all values in the constants (EXPLICIT_LICENSES and
+    # PACKAGE_REPO_OVERRIDES) appear in agent_requirements.in file
+
+    agent_requirements_path = app.repo.agent_requirements
```

Comment:
Let's abort here if the repo is not core

Reply:
Ah good catch, I forgot this validation is core-only!